### DISCUSSION TITLE
Save Pipe Position for Field Supply? (And combines with conveyor)

### DISCUSSION THREAD

**Discussion by MtBakerScum** [Courseplay/courseplay#7068] [Ideas]
would it be possible to add save pipe position to an auger wagon on a field supply course?

**Comment by Tensuko** [comment]
Sure that would make sence.

**Reply by Tensuko** [reply]
In Addition, also save for Combines with movable pipe.

**Reply by MtBakerScum** [reply]
Was that not already possible? I thought it was

**Reply by Tensuko** [reply]
No and thats why I added it here as reminder.

**Reply by MtBakerScum** [reply]
Since we're on the subject, what about adding a save tool height option for harvesters when users are running the tool height control mod? 

https://farming-simulator.com/mod.php?lang=en&country=it&mod_id=155882

**Reply by Tensuko** [reply]
That won't happen as it got no practical use, just visual.

**Reply by MtBakerScum** [reply]
It actually does keep headers from dragging on the ground and disrupting harvest speed

**Reply by MtBakerScum** [reply]
This video at the time stamp showcases its usefullness

https://youtu.be/77Bd5dPgPsc?t=438

**Reply by Tensuko** [reply]
It does not have an effect in the game, so nope.

**Reply by MtBakerScum** [reply]
It does though, thats demonstrated in the video I linked.  Some modded headers sit too low with some modded combines and drag on the ground, thus slowing down the harvester.

**Reply by Tensuko** [reply]
But thats modded combines and mostly use not the correct headers, thats why it is wrong, so still no.

**Reply by lxfTTjek** [reply]
Does cp conflict with the global company plugin

**Comment by Tensuko** [comment]
Will have this in the next release.

**Reply by MtBakerScum** [reply]
Will the return to work vehicle speed be addressed in the next release? My spreaders have been returning to the field stuck at 6mph or 10kph after refill

**Reply by rohne83** [reply]
@MtBakerScum the speed issue was solved with version 6.4.0.6

**Comment by MtBakerScum** [comment]
Yes, thank you
